pkg/jsx/jsx-runtime.js

```diff
@@ -0,0 +1,16 @@
+import { h, text } from 'hyperapp'
+
+const jsx = (type, { children = [], ...props }) =>
+  typeof type === 'function'
+    ? type(props || [], children)
```

Comment:
Why `props || []`? Props should be an object, no?

Reply:
Makes sense, not sure how I ended up with the array 🙊. I changed it to default to an object via parameter destructuring 👍